test/e2e/create_fallback_on_noop_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateFallbackOnNoop(t *testing.T) {
+	env := BuildEnv(t)
+	logger := Logger{}
+	kapp := Kapp{t, env.Namespace, env.KappBinaryPath, logger}
+
+	objNs := env.Namespace + "-create-fallback-on-noop"
+	nsYaml := strings.Replace(`
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: __ns__
+`, "__ns__", objNs, -1)
+	objYaml := strings.Replace(`
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: precious-resource
+  namespace: __ns__
```

Comment:
we can avoid specifying namespace and kapp will fill it with env configured ns.

Reply:
that's cool --  I cargo-culted it over from the test for the original fallback_on_... feature: https://github.com/vmware-tanzu/carvel-kapp/blob/develop/test/e2e/create_fallback_on_update_test.go#L31

I was wondering why it was important but didn't bother trying to remove it. :)